test-cases/gutenberg/writing-flow/rich-text-formatting.md

```diff
@@ -11,76 +11,11 @@ Have a rich-text based component with content (Paragraph, Heading, Quote, Media
 
 - [Text color selection is not preserved when cursor is placed in the middle of text](https://github.com/WordPress/gutenberg/issues/42714)
 
-**Bold, Italic, strikethrough buttons**
-
-- Press the Bold button and write a bold word.
-- Press the Italic button and write an italic word.
-- Press strikethrough button and write strikethrough text.
-
-**Highlight selected text**
-
-- Write some text and select it.
-- Tap on the Highlight text button (A).
-- Select a color.
-- Expect to see the text with the selected color.
-
 **Highlight text without selection**
 
 - Without selecting any text tap on the highlight text button (A).
 - Select a color and write some text.
 - Expect to see the newly introduced text with the previously selected color.
-
-
-##### TC002
-
-**Alignment buttons**
-
-- Press Left, Right, Center alignment buttons.
-- Check that the text responds accordingly.
-
-
-##### TC003
-
-**Alignment Split**
-
-- Write some lines on a paragraph block.
-- Set an alignment on the block.
-- Move the caret by the half of the text and press enter to split it.
-- Check that the new block has the same alignment.
-- Check that the text alignment corresponds to the setting.
-
-
-##### TC004
-
-**Link button works without selection**
-
-- Press the links button 
-- Fill the options and dismiss to add a link.
-- The link formatted word should be added to the block.
-
-
-##### TC005
-
-**Link button works with a selected word**
-
-- Write a word and select it.
-- Press the link format button.
-- Check that the `Link Text` field is pre-filled with the selected word.
-- Add a URL and dismiss.
-- The selected word now should be formatted as a link.
-
-
-##### TC006
-
-**Adding a link from a copied URL**
-
-- Go to a browser and copy a URL to the clipboard.
-- Back to Gutenberg, select a word on a rich-text based component.
-- Press the Link format button.
-- Check that the copied URL is pre-filled on the URL field.
-- Dismiss the bottom-sheet to see the selected word formatted as a link.
-
-
 ##### TC007
```

Comment:
Due to the removal of the above test cases, maybe we should the test case number to make it sequential. I.e. this should be `TC002`. WDYT?

Reply:
So far, we have not renumbered test cases, e.g. the current Columns cases includes [a single `10` case](https://github.com/wordpress-mobile/test-cases/blob/644026351243da07d3f40e0aff3d98d3407ccf6a/test-cases/gutenberg/columns.md#tc010). My thought is that while that practice is odd on its own, it is helpful when considering the context that we are incrementally moving away from these documented manual test cases.  

My preference would be to retain the numbers, not renumber them.